### PR TITLE
ENH Relative import

### PR DESCRIPTION
**Pull request by edouardoyallon**
Attempt to fix the import issues for circleci

### CODE REVIEW

**Comment by janden** [comment]
# [Codecov](https://codecov.io/gh/kymatio/kymatio/pull/579?src=pr&el=h1) Report
> Merging [#579](https://codecov.io/gh/kymatio/kymatio/pull/579?src=pr&el=desc) into [dev](https://codecov.io/gh/kymatio/kymatio/commit/8f27116cfe0d748e17f6262a2226057dbd668552&el=desc) will **decrease** coverage by `9.94%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/kymatio/kymatio/pull/579?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##              dev     #579      +/-   ##
==========================================
- Coverage   87.70%   77.75%   -9.95%     
==========================================
  Files          65       65              
  Lines        2343     2338       -5     
==========================================
- Hits         2055     1818     -237     
- Misses        288      520     +232     
```

| Flag | Coverage Δ | |
|---|---|---|
| #jenkins_main | `?` | |
| #travis | `77.75% <100.00%> (-0.01%)` | :arrow_down: |

| [Impacted Files](https://codecov.io/gh/kymatio/kymatio/pull/579?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [kymatio/backend/torch\_skcuda\_backend.py](https://codecov.io/gh/kymatio/kymatio/pull/579/diff?src=pr&el=tree#diff-a3ltYXRpby9iYWNrZW5kL3RvcmNoX3NrY3VkYV9iYWNrZW5kLnB5) | `0.00% <0.00%> (-100.00%)` | :arrow_down: |
| [...matio/scattering3d/backend/torch\_skcuda\_backend.py](https://codecov.io/gh/kymatio/kymatio/pull/579/diff?src=pr&el=tree#diff-a3ltYXRpby9zY2F0dGVyaW5nM2QvYmFja2VuZC90b3JjaF9za2N1ZGFfYmFja2VuZC5weQ==) | `0.00% <0.00%> (-100.00%)` | :arrow_down: |
| [...matio/scattering1d/backend/torch\_skcuda\_backend.py](https://codecov.io/gh/kymatio/kymatio/pull/579/diff?src=pr&el=tree#diff-a3ltYXRpby9zY2F0dGVyaW5nMWQvYmFja2VuZC90b3JjaF9za2N1ZGFfYmFja2VuZC5weQ==) | `0.00% <0.00%> (-97.30%)` | :arrow_down: |
| [...matio/scattering2d/backend/torch\_skcuda\_backend.py](https://codecov.io/gh/kymatio/kymatio/pull/579/diff?src=pr&el=tree#diff-a3ltYXRpby9zY2F0dGVyaW5nMmQvYmFja2VuZC90b3JjaF9za2N1ZGFfYmFja2VuZC5weQ==) | `0.00% <0.00%> (-100.00%)` | :arrow_down: |
| [kymatio/backend/torch\_backend.py](https://codecov.io/gh/kymatio/kymatio/pull/579/diff?src=pr&el=tree#diff-a3ltYXRpby9iYWNrZW5kL3RvcmNoX2JhY2tlbmQucHk=) | `91.89% <0.00%> (-6.76%)` | :arrow_down: |
| [kymatio/scattering1d/backend/torch\_backend.py](https://codecov.io/gh/kymatio/kymatio/pull/579/diff?src=pr&el=tree#diff-a3ltYXRpby9zY2F0dGVyaW5nMWQvYmFja2VuZC90b3JjaF9iYWNrZW5kLnB5) | `97.29% <0.00%> (-2.71%)` | :arrow_down: |
| [kymatio/scattering3d/backend/torch\_backend.py](https://codecov.io/gh/kymatio/kymatio/pull/579/diff?src=pr&el=tree#diff-a3ltYXRpby9zY2F0dGVyaW5nM2QvYmFja2VuZC90b3JjaF9iYWNrZW5kLnB5) | `95.23% <0.00%> (-1.59%)` | :arrow_down: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/kymatio/kymatio/pull/579?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/kymatio/kymatio/pull/579?src=pr&el=footer). Last update [8f27116...4780d50](https://codecov.io/gh/kymatio/kymatio/pull/579?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by edouardoyallon** [comment]
Please merge!

**Comment by edouardoyallon** [comment]
So: what should we do?(I'm ok with both standard as we don't mix two different styles)

File: kymatio/frontend/tensorflow_frontend.py [hunk 1]
```diff
@@ -1,6 +1,4 @@
 import tensorflow as tf
-from packaging import version
```
Comment (eickenberg) [inline kymatio/frontend/tensorflow_frontend.py]: what does this have to do with relative import?

File: kymatio/scattering2d/backend/numpy_backend.py [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 import numpy as np
 from collections import namedtuple
-from scipy.fftpack import fft2, ifft2
```
Comment (eickenberg) [inline kymatio/scattering2d/backend/numpy_backend.py]: what does this have to do with relative imports

File: kymatio/scattering2d/backend/tensorflow_backend.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # Authors: Edouard Oyallon, Sergey Zagoruyko, Muawiz Chaudhary
 
 import tensorflow as tf
-import numpy as np
```
Comment (eickenberg) [inline kymatio/scattering2d/backend/tensorflow_backend.py]: what does this have to do with relative imports

File: kymatio/scattering2d/backend/torch_skcuda_backend.py [hunk 4]
```diff
@@ -2,7 +2,6 @@
 
 from collections import namedtuple
 import torch
-from skcuda import cublas
```
Comment (eickenberg) [inline kymatio/scattering2d/backend/torch_skcuda_backend.py]: what does this have to do with relative imports

File: kymatio/scattering3d/backend/numpy_backend.py [hunk 5]
```diff
@@ -1,6 +1,4 @@
 import numpy as np
-import warnings
```
Comment (eickenberg) [inline kymatio/scattering3d/backend/numpy_backend.py]: what does this have to do with relative imports

**Comment by edouardoyallon** [comment]
@eickenberg I corrected them - I think it'd be worth to remove the library i noticed were extra-included

**Comment by eickenberg** [comment]
tests ran through locally as well.
Not sure why codecov is unhappy, but it doesn't seem important.
So I think this can be merged by rebase

**Comment by edouardoyallon** [comment]
@eickenberg codecov is unhappy coz of the covererage decrease.
